Entries/Copy-X/Copy-X.md

```diff
@@ -0,0 +1,283 @@
+[^D_2182905]: [Manfred Mann* – 2006](https://www.discogs.com/release/2182905-Manfred-Mann-2006)
+[^D_451419]: [De-Phazz – Godsdog](https://www.discogs.com/release/451419-De-Phazz-Godsdog)
+
+#### copy-X audio:
+
+No samples have been tested at the moment, so exact specifics cannot be confirmed. Used to protect Audio CDs.
+
+#### copy-X trial maker:
+
+No samples currently known, and thus no information either.
+
+
+***
+
+## Samples:
+ 
+#### Confirmed:
+
+* copy-X light[^84759][^107929][^81628][^Kenny][^101786]
+* copy-X professional[^108150][^48393][^82475][^DE_EM2DX][^Petterson]
+
+#### Unconfirmed: 
+
+* copy-X audio[^D_8232013][^D_1436646][^D_2182905][^D_451419]
+* German Westerner 2[^pcgames_de_westerner2][^gamestar_de_westerner2]
+* Many games from Sixteen Tons Entertainment
+* Many games from Tivola
+* Many games from Terzio
+* Many pieces of software from Magix 
+
+[^pcgames_de_westerner2]: [The Westerner 2 im PC-Games-Test: "Ein enttäuschendes Adventure ... " ](https://www.pcgames.de/The-Westerner-2-Fenimore-Fillmores-Revenge-Spiel-38466/Tests/The-Westerner-2-im-PC-Games-Test-Ein-enttaeuschendes-Adventure-679282/)
+[^gamestar_de_westerner2]: [The Westerner 2: Westernheld Fenimore Fillmore ist er-
+wachsen geworden. Im Gegenzug
+schrumpft die Spielzeit auf Babyformat.](https://download.gamestar.de/public/43400/43452/05_2009_096.pdf)
+
+***
+
+## Supported Platforms:
+
+* light: Any (no software protection is employed, although it appears to mainly be marketed for Windows games regardless.)
+
+* professional: Windows
+(Some professional games are Hybrid PC/Mac discs, but the Mac versions of these games do not appear to utilize any sort of software protection.)[^108150]
+
+* audio: Audio CD
+
+***
+
+## Software Known to Detect:
+
+* ProtectionID
+* BinaryObjectScanner
+
+***
+
+## Preservation Instructions:
+
+light: None
+
+professional: Any sort of basic copy of the disc image appears to work.
+
+*** 
+
+## Associated File Attributes:
+
+### File Directories:
+
+#### "System":
+Directory for dummy files used by most professional discs so far, as well as some light[^Kenny] discs.
+
+#### "ZDAT":
+Directory for dummy files used by some, but not all light discs, and one known professional[^Petterson] disc so far. One calls it zdata[^108150], at least one uses System instead. Further variance likely.
+
+### File Names:
+Note: file names are not case sensitive, and vary from disc to disc.
+#### "gov_*.x64":
+Dummy file associated with professional. Names include gov_io.x64, gov_05.x64, and mov_io.x64.
+
+#### "iofile.x64":
+Dummy file associated with professional. Seems to be directly referenced by optgraph.dll.
+
+#### "sound.x64":
+Dummy file associated with professional. Seemingly not present on at least one professional disc.
+
+#### "optgraph.dll":
+Dll file responsible for software protection on professional discs.
+
+***
+
+## Known Manufacturing Information:
+
+### Known Manufacturing Plants:
+
+* optimal media production
```

Comment:
Please add the context that we know this should be the only manufacturing plant producing these discs.

Reply:
Maybe fixed?

Reply:
Confirmed fixed.